docs/source/ProgrammingGuide/Compiling.md

```diff
@@ -255,7 +244,6 @@ A word of caution on where to include the `Makefile.kokkos`: since the embedded
 
 More example application Makefiles can be found in the tutorial examples under `kokkos/example/tutorial`.
```

Comment:
Isn't line 245 a bit conflict-y with steering people away from using Makefiles?  IOW, don't do anything to encourage it, if that's not what you want users to do.